course-definition.yml

```diff
@@ -118,6 +118,14 @@ marketing:
         I think the instant feedback right there in the git push is really cool.
         Didn't even know that was possible!
 
+extensions:
```

Comment:
is there no way to limit extension visibility, like alpha or beta?

Reply:
Not yet, but I have a workaround - @sarp I've created an account with a membership for you on staging, as long as we don't "apply" the update on production and only on staging, we should be good. 

Link: https://app-staging.codecrafters.io/courses/bittorrent/admin/updates